honcho-cli/src/honcho_cli/commands/session.py

```
@@ -0,0 +1,407 @@
+
+        items = [
+            {
+                "id": s.id,
+                "is_active": s.is_active,
+                "metadata": s.metadata,
+                "created_at": str(s.created_at),
+            }
+            for s in raw_sessions
+        ]
+        print_result(items, columns=["id", "is_active", "metadata", "created_at"], title="Sessions")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)